src/main/java/com/github/wovnio/wovnjava/PatternHandler.java

```diff
@@ -0,0 +1,33 @@
+package com.github.wovnio.wovnjava;
+
+import java.util.regex.Pattern;
+import java.util.regex.Matcher;
+
+abstract class PatternHandler {
+    protected Pattern getLangPattern;
```

Comment:
I think `langRegex` is enough and will avoid the weirdness of `getGetLangPattern`. But since is it likely temporary, I guess it is not a huge deal.

Reply:
It was originally called `langPattern`, but afterwards patterns for remove lang and insert lang were needed, so I renamed it. I agree, `getGet...` is not great. It should be temporary, however.

Reply:
I have removed outside access to `getLangPattern`, so there is no longer need for the wonky getter.

Reply:
:+1: